annotator_store/views.py

```diff
@@ -236,6 +236,17 @@ def get(self, request):
                 )
                 # NOTE: contains search on extra data jsonfield is
                 # probably not a great idea...
+            # non-standard field search on extra data
+            # ignore limit and offset field because they must be used afterward
+            elif field not in ['limit','offset']:
+                # evaluate queryset for extra_data JSONField parsing
+                annotations = notes.all()
```

Comment:
Am I following this correctly that you're pulling all the annotations down and filtering them locally?  That won't scale.   Could you look at a solution like some of the suggestions here? https://stackoverflow.com/questions/10445176/how-to-write-a-query-to-get-find-value-in-a-json-field-in-django

Reply:
Yes, that won't scale. The __contains solution is better. I didn't think about using it with json format. In a way it's searches exactly that value, not what usually contains do. I'm going to implement it!